docs/xdump.md

```diff
@@ -106,6 +106,7 @@ An exception is the "tool" agent, which can be used to trigger external processe
 | snap                      | Take a snap of the trace buffers, which contain tracepoint data.                                       |
 | ceedump                   | LE CEEDUMP dumps are z/OS&reg; formatted summary system dumps that show stack traces for each thread that is in the VM process, together with register information and a short dump of storage for each register. |
 | jit                       | JIT compiler dumps contain diagnostic data in a binary format.                                         |
+| exit                       | Shutdown the JVM.                                                                                     |
```

Comment:
> * Does this need to cross-refer to `[-XX:[+|-]ExitOnOutOfMemory](xxexitonoutofmemory.md)`?

Same as above. While its used in the implementation of -XX:[+|-]ExitOnOutOfMemory it may have other uses as well.

Reply:
Those types of links won't work from here Theresa as this page is not built as part of the use doc build chain. 

There are probably lots of cases for adding links throughout this build tool. I don't think we have any currently. We'll discuss this with Paul and do them all at once, if we decide it's appropriate.